### DISCUSSION TITLE
[ FF7 ] Crash After Midgar Zolom Cutscene

### DISCUSSION THREAD

**Discussion by Bliskknight** [julianxhokaxhiu/FFNx#250] [Q&A] [answered]
Ran into a crash after the Midgar Zolom cutscene when leaving the area. Not sure what is the cause of it. Attaching the crash report. I am using 7th heaven and the 2.0 new threat mod.

[ff7 crash.txt](https://github.com/julianxhokaxhiu/FFNx/files/6528811/ff7.crash.txt)

It doesn't seem very usable though. It won't upload in original form.

**Comment by julianxhokaxhiu** [comment, marked as answer]
Hi, thanks for reporting this but when you use 7th Heaven and in general anything that is not vanilla FFNx as it comes, support here is non existant. Please consider using 7th Heaven support channels instead. Cheers.

**Reply by Bliskknight** [reply]
Sounds good, I'm new to this and it told me to report here. Thanks for the quick response!

**Reply by julianxhokaxhiu** [reply]
No worries, you're welcome :)
